helpers/controllers/authController.test.js

```diff
@@ -81,4 +89,75 @@ describe('auth controller', () => {
       success: true
     })
   })
+
+  test('Signup - should resolve with success false if userinfo is incomplete', async () => {
+    const result = await signup({}, {}, { req: { session: {} } })
+    expect(result).toEqual({
+      success: false,
+      error: 'Invalid registration information'
+    })
+  })
+
+  test('Signup - should resolve with success false if user already exists', async () => {
+    db.User.findOne = jest.fn().mockReturnValue({ username: 'c0d3user' })
```

Comment:
Should this not return `username: 'testuser'` to match `userArgs`?

Reply:
I guess since you mock it the return value in itself doesn't matter as long as something is returned?

Reply:
It doesn't matter, it just have to return some user. A nitpick would be to return the same username though, it'll make prevent questions like this in the future.

Reply:
`userArgs` is setup to be a valid entry for signup. In this case, I wanted to make sure it would error out so all the fields aren't filled out.